client/src/components/SelectTabs/EventTypesTab.js

```diff
@@ -296,7 +296,7 @@ export default function EventTypesTab() {
       .post("/api/event", {
         ...eventBody,
         duration: minutes,
-        link: `${userURL}/${eventBody.link}`,
+        link: encodeURI(`${userURL}/${eventBody.link}`),
```

Comment:
The proper function should be encodeURIComponent I think. But this should work as well.

Reply:
Link isn't a complete uri, it's just the user's url pref and suffix so this should be fine I believe!